README.rst

```diff
@@ -0,0 +1,114 @@
+=====================================
+Diffusion Equation with Linear Source
+=====================================
+
+This example solves the weak form of the following diffusion equation with a source term,
+
+|diffusion_equation|
+
+using the Galerkin Finite Element method. |conductivity_tensor| , |alpha| and |psi| are the positive definite and symmetric rank two conductivity tensor, a scalar parameter (e.g. thermal capacity) and a function of the dependent variable |u| (linear in this example) respectively. The dependent variable |u| is a spatially varying scalar field (e.g. temperature). In this example an isotropic and homogeneous material with |equation1| (identity tensor) and |equation2| is considered. 
+
+The function or source term takes the following linear form.
+
+|equation3|
+
+In general, parameters |a| and |b| are functions of the independent variable |x|, but since the material is homogeneous (no spatially varying physical properties), they remain unchanged throughout the domain. 
+
+Note that boundary conditions are prescribed to match those required for the analytical solution.
+
+.. |diffusion_equation| image:: ./docs/images/diffusion_equation_with_source.svg 
+   :align: middle
+
+.. |conductivity_tensor| image:: ./docs/images/conductivity_tensor.svg 
+   :align: middle
+ 
+.. |alpha| image:: ./docs/images/alpha.svg 
+   :align: bottom
+   
+.. |psi| image:: ./docs/images/psi.svg 
+   :align: bottom
+   
+.. |u| image:: ./docs/images/u.svg 
+   :align: bottom
+   
+.. |equation1| image:: ./docs/images/equation1.svg 
+   :align: middle
+  
+.. |equation2| image:: ./docs/images/equation2.svg 
+   :align: bottom
+   
+.. |equation3| image:: ./docs/images/equation3.svg 
+   :align: middle
+   
+.. |a| image:: ./docs/images/a.svg  
+   :align: bottom
+
+.. |b| image:: ./docs/images/b.svg  
+   :align: bottom
+   
+.. |x| image:: ./docs/images/x.svg  
+   :align: bottom   
+   
+   
+Building the example
+====================
+
+The fortran version of the example can be configured and built with CMake::
+
+  git clone https://github.com/OpenCMISS-Examples/diffusion_equation_with_linear_source
+  mkdir diffusion_equation_with_linear_source-build
+  cd diffusion_equation_with_linear_source-build
+  cmake -DOpenCMISSLibs_DIR=/path/to/opencmisslib/install ../diffusion_equation_with_linear_source
```

Comment:
-DOpenCMISSLibs_DIR=/path/to/opencmisslibs/install

Added an 's'.

Reply:
This example doesn't have a python script. The four examples I have committed so far were existing examples which I modified conforming to the new format. Let me know if you want me get rid of the statement on python version. I hope we can include python version sometime in the future. Also note that the previous example "diffusion_equation" also did not have the python script.